File: 프로그래머스/1/12933. 정수 내림차순으로 배치하기/정수 내림차순으로 배치하기.java
import java.util.Arrays;

class Solution {
    public long solution(long n) {
        char[] arr = Long.toString(n).toCharArray();
        
        Arrays.sort(arr);
        
        String str = "";
        
        for (int i = arr.length - 1; i >= 0; i--) {
            str += arr[i];
        }
        
        long answer = 0;
        
        answer = Long.parseLong(str);
        
        return answer;
    }
}

File: 프로그래머스/1/12933. 정수 내림차순으로 배치하기/정수 내림차순으로 배치하기.py
def solution(n):
    answer = ""
    temp = []
    s = str(n)
    
    for i in s :
        temp.append(int(i))
    temp = sorted(temp, reverse=True)
    
    for i in temp :
        answer += str(i)
    
    return int(answer)